### PR TITLE
Innkeeper emails addresses

### PR DESCRIPTION
**Pull request by loneil**
Adjust innkeeper inbox addresses
Set dev reservation password timeout to longer.

### CODE REVIEW

File: charts/tenant-ui/values-e79518-dev.yaml [hunk 1]
```diff
@@ -24,8 +24,8 @@ tenant_ui:
   smtp:
     server: apps.smtp.gov.bc.ca
     port: 25
-    senderAddress: DoNotReplyTractionPR@gov.bc.ca
-    innkeeperInbox: lucas.oneil@gov.bc.ca
+    senderAddress: DoNotReplyTractionDTS@gov.bc.ca
+    innkeeperInbox: emiliano.sune@quartech.com
```
Comment (loneil) [inline charts/tenant-ui/values-e79518-dev.yaml:28]: @esune think this is the values file for the DTS deployment? Is it used or did we go another way?

File: charts/tenant-ui/values-development.yaml [hunk 2]
```diff
@@ -21,7 +21,7 @@ tenant_ui:
     server: apps.smtp.gov.bc.ca
     port: 25
     senderAddress: DoNotReplyTractionDEV@gov.bc.ca
-    innkeeperInbox: jason.sherman@gmail.com
+    innkeeperInbox: "lucas.o'neil@gov.bc.ca,emiliano.sune@quartech.com"
```
Comment (loneil) [inline charts/tenant-ui/values-development.yaml:24]: @esune let me know if you don't want to be on here for the dev env, or if we want other innkeeper contacts or anything

File: charts/traction/values-development.yaml [hunk 3]
```diff
@@ -15,7 +15,7 @@ acapy:
     tractionInnkeeper:
       printKey: true
       printToken: true
-      reservationExpiryMinutes: 2880
+      reservationExpiryMinutes: 7200
```
Comment (loneil) [inline charts/traction/values-development.yaml:18]: 5 days

File: charts/traction/values-pr.yaml [hunk 4]
```diff
@@ -21,7 +21,7 @@ acapy:
       printKey: true
       printToken: true
       reservationExpiryMinutes: 2880
-      reservationAutoApprove: true
+      reservationAutoApprove: false
```
Comment (loneil) [inline charts/traction/values-pr.yaml:24]: Left this true on a previous PR. Might be useful to have auto approve for pull requests, but also might want to adjust reservation flow and test it in a PR 🤷‍♂️ can alter on a case-by-case basis when needed

File: charts/tenant-ui/values-e79518-dev.yaml (hunk 1 above)
Comment (esune) [reply charts/tenant-ui/values-e79518-dev.yaml:28]: It is used, but will get decommissioned soon (hopefully). Proposed new values are good 👍🏻

File: charts/tenant-ui/values-development.yaml (hunk 2 above)
Comment (esune) [reply charts/tenant-ui/values-development.yaml:24]: I'm fine being there. This env will be re-configured soon anyway, and having a backup for approvals seems like a good idea.

**Review by esune — COMMENTED** [review]
:+1

**Review by esune — APPROVED** [review]

**Pull request merged**